core/whistle_couch-1.0.0/src/couch_util.erl

```diff
@@ -643,7 +649,7 @@ prepare_doc_for_del(Conn, #db{name=DbName}, Doc) ->
         [{<<"_id">>, Id}
          ,{<<"_rev">>, DocRev}
          ,{<<"_deleted">>, 'true'}
-         ,{<<"pvt_type">>, wh_json:get_value(<<"pvt_type">>, Doc)}
+         | publish_fields(Doc)
```

Comment:
I wonder how we can avoid this, if at all? This is the tombstone document so I was trying to minimize the number of fields we store in it to save space in the DBs.